examples/support/observability/Makefile

```diff
@@ -1,20 +1,12 @@
 # renovate: datasource=github-tags depName=jaegertracing/jaeger
 JAEGER_VERSION ?= v1.41.0
 # renovate: datasource=github-tags depName=cert-manager/cert-manager
-CERT_MANAGER_VERSION ?= v1.10.1
```

Comment:
was this removal intended? the renovate comment is still there

Reply:
yes, it was ... will remove the comment